firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -108,7 +108,7 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
 
     func tapKeyboardKey(_ key: Int) {
         let key = app.keyboards.keys.element(boundBy: key)
-        if app.buttons["Continue"].exists == true {
+        if app.buttons["Continue"].isHittable {
```

Comment:
I have had many issues with this .isHittable.., let's monitor that tests work (I see you added a link to a build working, but let's keep an eye on this)

Reply:
The "Continue" button seems to exist but is covered, so `tap()` within the if-statement does not work at least on my end.